src/isUndefined.ts

```diff
@@ -1,3 +1,3 @@
 // the lodash way https://github.com/lodash/lodash/blob/master/isUndefined.js
 
-export const isUndefined = (_: unknown): boolean => _ === undefined;
+export const isUndefined = (_: unknown): e is undefined => _ === undefined;
```

Comment:
should it be `_` rather than `e`?
```suggestion
export const isUndefined = (_: unknown): _ is undefined => _ === undefined;
```

Reply:
🙄